addons/mail/static/src/discuss/core/common/discuss_core_common_service.js

```diff
@@ -118,9 +118,12 @@ export class DiscussCoreCommon {
                 thread: { id: channel_id, model: "discuss.channel" },
             });
             if (member?.persona.eq(this.store.self)) {
-                member.thread.updateSeen(
-                    last_message_id ? this.store.Message.get(last_message_id) : null
-                );
+                if ("force_local_seen_message" in payload) {
```

Comment:
`force_local_seen_message` can be renamed to `force_new_message_separator`, to follow naming convention for everything related to new message separator.